RecoPixelVertexing/PixelLowPtUtilities/bin/ClusterShapeAnalyzer.cc

```diff
@@ -0,0 +1,402 @@
+#include <cmath>
+
+#include <utility>
+#include <vector>
+#include <map>
+#include <fstream>
+#include <iostream>
+
+using namespace std;
+
+#include "TROOT.h"
+#include "TFile.h"
+#include "TH2F.h"
+#include "TH1F.h"
+#include "TNtuple.h"
+
+//#include "gnuplot_i.hpp"
+//Gnuplot * g;
+
+// Total loss, loss in one direction is Loss/2 
+#define Loss 0.005
+
+// FIXME
+#define MinEntries 100
+
+TH1D * line;
+
+#define exMax 10
+#define eyMax 15
+
+const char *subName[2] = {"barrel","endcap"};
```

Comment:
Sorry, ClusterShapeAnalyzer is very old legacy code... It works, but for sure its code quality is far from being perfect. However, if it is the only place (!) where static analyzer generates a warning, it could be fixed fast.